### ISSUE TITLE
Hooks only trigger on build of master branch

### ISSUE THREAD

**Issue by philip-iv** [tcplugins/tcWebHooks#69]
I'd like the webhooks to trigger on every build, but currently they're only triggered on builds of master

**Comment by netwolfuk** [comment]
It should be working for all branches.
Can you post your tcWebHooks version, TeamCity version and the webhook config from plugin-settings.xml.
Thanks.

**Comment by netwolfuk** [comment]
Closed due to no response.

**Comment by philip-iv** [comment]
Sorry, was waiting on info from admins - TC 10.0.4, plugin 0.9. This is the only trigger filter configured:
`<filter value="${branchDisplayName}" regex="^branch-xyz$" enabled="true"/>`

**Comment by netwolfuk** [comment]
Thanks. I'll take a look and see if I can reproduce it.

**Comment by netwolfuk** [comment]
Looking at the code, the filter support was never merged into the 0.9 version. I'll take a look at how hard it will be. At the time 1.1 was nearly ready, but that was over a year ago now, and life has gotten in the way unfortunately.

**Comment by philip-iv** [comment]
So upgrading to 1.1alpha will give the needed functionality?

**Comment by netwolfuk** [comment]
Yes. That's correct.

**Comment by netwolfuk** [comment]
BTW. I had a very quick look through the commits. It should be possible to merge into master if you decide you can't use an alpha.

**Comment by philip-iv** [comment]
Alpha works fine for me. Thanks for the help. I would suggest mentioning on the documentation that the filtering is only a feature in alpha, not the main build.

**Comment by netwolfuk** [comment]
Thanks for the feedback and sorry for the confusion. I will update the docs.

**Comment by netwolfuk** [comment]
Wiki updated: https://github.com/tcplugins/tcWebHooks/wiki/Advanced-Editing-of-tcWebHooks-configuration
